docs/app/Examples/collections/Menu/Variations/ColoredInvertedMenus.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react'
+import { Menu } from 'stardust'
+
+class ExampleMenu extends Component {
+  static propTypes = {
+    color: PropTypes.string,
+  }
+
+  state = { activeItem: 'home' }
+
+  handleItemClick = (e, { name }) => this.setState({ activeItem: name })
+
+  render() {
+    const { color } = this.props
+    const { activeItem } = this.state
+
+    return (
+      <Menu color={color} inverted widths={3}>
+        <Menu.Item
+          name='home'
+          active={activeItem === 'home'}
+          onClick={this.handleItemClick}
+        >
+          Home
+        </Menu.Item>
+
+        <Menu.Item
+          name='messages'
+          active={activeItem === 'messages'}
+          onClick={this.handleItemClick}
+        >
+          Messages
+        </Menu.Item>
+
+        <Menu.Item
+          name='friends'
+          active={activeItem === 'friends'}
+          onClick={this.handleItemClick}
+        >
+          Friends
+        </Menu.Item>
+      </Menu>
+    )
+  }
+}
+
+const ColoredInvertedMenus = () => {
+  const colors = Menu._meta.props.color
+  const menus = colors.map(color => <ExampleMenu color={color} key={color} />)
+
+  return <div>{menus}</div>
+}
+
+export default ColoredInvertedMenus
```

Comment:
The loop is perfectly acceptable, but let's not show use of the private `_meta` prop.  I don't want users knowing/using that.  I think putting it in the docs will encourage use of it.  Instead, lets' just list the colors in an array explicitly.